activitysim/core/configuration/logit.py

```diff
@@ -77,6 +77,27 @@ class BaseLogitComponentSettings(PydanticCompute):
     CONSTANTS: dict[str, Any] = {}
     """Named constants usable in the utility expressions."""
 
+    preprocessor: PreprocessorSettings | list[PreprocessorSettings] | None = None
+    """Chooser preprocessor settings.
+
+    This is a set of expressions to be evaluated on the choosers
+    before the logit model is run. It is used to prepare the choosers
+    for the logit model by adding columns that are used in the
+    utility expressions.
+    """
+
+    annotate_households: PreprocessorSettings | None = None
```

Comment:
For autodoc, have each annotate_* field have its own docstring.

Reply:
done